kodilla-testing/src/main/java/com/kodilla/testing/library/LibraryUser.java

```diff
@@ -0,0 +1,51 @@
+package com.kodilla.testing.library;
+
+import java.util.Objects;
+
+public class LibraryUser {
+    private String firstname;
+    private String lastname;
```

Comment:
pola finalne :) wszędzie gdzie możesz dodawać final to dodawaj. masz dzięki temu większą kontrolę i pewność, że nie zmieni Ci się instancja serwisu/klasy, której używasz